clang/unittests/Format/FormatTest.cpp

```diff
@@ -16815,6 +16817,18 @@ TEST_F(FormatTest, ConfigurableSpacesInParens) {
   verifyFormat("void __attribute__((naked)) foo(int bar)", Spaces);
   verifyFormat("void f( ) __attribute__((asdf));", Spaces);
 
+  Spaces.SpacesInParensOptions.InAttributeSpecifiers =
+      FormatStyle::SIPCS_Always;
+  verifyFormat("SomeType *__attribute__( ( attr ) ) *a = NULL;", Spaces);
+  verifyFormat("void __attribute__( ( naked ) ) foo(int bar)", Spaces);
+  verifyFormat("void f( ) __attribute__( ( asdf ) );", Spaces);
+  Spaces.SpacesInParensOptions.InAttributeSpecifiers =
```

Comment:
```suggestion
  verifyFormat("void f( ) __attribute__( ( asdf ) );", Spaces);
  
  Spaces.SpacesInParensOptions.InAttributeSpecifiers =
```
For a better readability.